### ISSUE TITLE
The "Split css" commit from Jul 18 has caused our images to exceed window width

### ISSUE THREAD

**Issue by FrankReh** [g1eny0ung/hugo-theme-dream#140]
We built a hugo server on our home network earlier this year and had found your theme to be very nice, so thank you.

Question: your commit 9acb03e649540a289d794f6c1e06d4af511181f2 named "Split css" has changed the behavior of how our images are displayed. Wonder if we're doing something wrong. Now our jpg images are displayed at full resolution rather than being sized to fit the width of the desktop window or the mobile screen. They used to be sized to fit the available view port width.

We run the hugo server behind our home firewall so can't show the results, sorry. It's a project for the family. The pages all have the same problem for us. Each typically has a reference to a jpg file like
`![Picture name](/img/photo.jpg)`

I can see the file changes this commit made, but I don't understand hugo well enough to know where to start in narrowing things down easily. Can you offer a suggestion?

TIA.

**Comment by g1eny0ung** [comment]
Can you provide me with a screenshot? You can hide your private information with a mosaic. In fact, if the resolution of the image in the post is too large, it will adapt to the width of the outer container, that is, it will not be wider than the width of the article area.

**Comment by FrankReh** [comment]
The image is honoring the border on the left side of the browser window but not the right side, or the bottom for that matter. I've seen the same with Chrome on my desktop and Safari on my desktop and phone.

<img width="936" alt="Screen Shot 2020-08-06 at 1 56 23 PM" src="https://user-images.githubusercontent.com/5726008/89566464-36cfd580-d7ee-11ea-8057-b64313252e66.png">

**Comment by g1eny0ung** [comment]
This problem did occur a long time ago, I remember I have fixed it. Is the theme you used is the latest?

**Comment by FrankReh** [comment]
Oh yes. Totally my fault. I just remembered I had created a copy of your _default/single.html and put it into the top level layouts/_default/single.html. When I removed my local copy, your theme worked again as expected from the latest tip.

Sorry about that. I'll make a fresh copy and work in my changes again. (I had removed the save button and added a print button I think.)

Thanks again and thanks for a very nice theme.
